twitchio/ext/eventsub/models.py

```diff
@@ -82,33 +89,104 @@ def __init__(self, request: web.Request):
         self._raw_timestamp = request.headers["Twitch-Eventsub-Message-Timestamp"]
 
 
+class WebsocketHeaders:
+    """
+    The headers of the inbound Websocket EventSub message
+
+    Attributes
+    -----------
+    message_id: :class:`str`
+        The unique ID of the message
+    message_type: :class:`str`
+        The type of the message coming through
+    message_retry: :class:`int`
+        Kept for compatibility with :class:`Headers`
+    signature: :class:`str`
+        Kept for compatibility with :class:`Headers`
```

Comment:
Real query,
Do the docs show correctly for `Headers` here?
I ask because I personally have the issue with Sphinx where it won't understand the directive since it's not akin to `currentmodule:: ...`

Reply:
The currentmodule is set to twitchio.ext.eventsub, so it should. The WebsocketHeaders didnt make it into the docs though, so ill fix that